### PR TITLE
[WIP] Support BLIP and GIT in image-to-text and VQA pipelines

### LINKED ISSUE #21110: Add support for BLIP and GIT in image-to-text and VQA pipelines
### Feature request

BLIP and GIT are 2 recent additions in the library, providing state-of-the-art performance for tasks like image captioning and visual question answering (VQA). GIT is even capable of video captioning and video QA.

Hence it makes sense to support them in our image-to-text and VQA pipelines.

### Motivation

Having support for better models in pipelines is very desired!

See also a request for it here: https://discuss.huggingface.co/t/support-for-different-models-in-text-to-image-pipeline/29504

### Your contribution

I can assist in adding support, see #18446 as a very similar case

### PR DESCRIPTION
**Pull request by atturaioe**
# What does this PR do?
Support BLIP and GIT models in image-to-text and VQA pipelines.

Fixes #21110 

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@NielsRogge

### CODE REVIEW

File: src/transformers/__init__.py [hunk 1]
```diff
@@ -778,7 +778,7 @@
     _import_structure["image_utils"] = ["ImageFeatureExtractionMixin"]
     _import_structure["models.beit"].extend(["BeitFeatureExtractor", "BeitImageProcessor"])
     _import_structure["models.bit"].extend(["BitImageProcessor"])
-    _import_structure["models.blip"].extend(["BlipImageProcessor"])
+    _import_structure["models.blip"].extend(["BlipFeatureExtractor", "BlipImageProcessor"])
```
Comment (NielsRogge) [inline src/transformers/__init__.py:781]: No need to add the feature extractor, you can use the image processor for the pipeline.

Comment (atturaioe) [reply src/transformers/__init__.py:781]: Yes, but the pipeline still builds a feature_extractor(which is actually a blipimageprocessor)?

**Comment by atturaioe** [comment]
Now, I need somehow to get the top_k return sequences for the VQA pipeline with generation.

File: src/transformers/__init__.py (hunk 1 above)
Comment (NielsRogge) [reply src/transformers/__init__.py:781]: Now not anymore ;) please rebase on main

**Pull request closed without merging**